test/e2e/images_test.go

```diff
@@ -240,6 +240,7 @@ WORKDIR /test
 	})
 
 	It("podman pull by digest and list --all", func() {
+		Skip("FIXME-8165: 'rmi -af' fails with 'layer not known'")
```

Comment:
This one makes me deeply uncomfortable - it's that easy to break our storage in such a way that we can't fix it with `rmi`?

Reply:
This was much, much harder to reproduce than I had expected. Added script to https://github.com/containers/podman/issues/6510#issuecomment-717967486